app/components/sessions/SessionsController.js

```diff
@@ -1,107 +1,119 @@
+                                    sc.Sessions[i] = [];
+                                }
+                                $sessionStorage.sessionset = sc.Sessions;
+
+                                for (var j = 0; j < response.data.sessions.length; j+= 1) {
+                                    var dayDiff = DateUtils.DateDiff.inDays(
+                                        $sessionStorage.event.begin,
+                                        response.data.sessions[j].begin);
+                                    //Filter out any mistakenly entered sessions outside date range
+                                    if (dayDiff>openevent.totalDays || dayDiff < 0) {
+                                        console.log('Session date = ' + dayDiff
+                                            +' outside event date range = '
+                                            + openevent.totalDays);
+                                        continue;
+                                    }
+                                    sc.Sessions[dayDiff].push(response.data.sessions[j]);
+                                    $sessionStorage.days[dayDiff].sessions = sc.Sessions[dayDiff];
+                                    $sessionStorage.days[dayDiff].sessions.sort(SortUtils.sortBy(
+                                    	'begin',
+                                    	false,
+                                    	function(a){return a;}
+                                    	));
+                                }
+                                sc.showLoaders = false;
+                                sc.Days = $sessionStorage.days;
+
+                            }, function (error) {
+                                console.error(error);
+                            });
+                    }
+                    sc.duration = function(session) {
+                        var start = DateUtils.getHourMin(session.begin);
+                        var end = DateUtils.getHourMin(session.end);
+
+                        return {start: start, end: end};
+                    };
+                    sc.showSession = function(session, event) {
+                        $mdDialog.session = {
+                            singleSession: session
+                        };
+                        $mdDialog.show({
+                            controller: 'SessionDialogController',
+                            templateUrl: 'app/components/sessions/sessiondialog.html',
+                            parent: angular.element(document.body),
+                            targetEvent: event,
+                                      });
+                    };/*Display only when id match */
+                    sc.display=function(session,event){
+                                 for(var i=0;i<sc.Days.length;i++){
+                                 for(var x in sc.Days[i].sessions){
+                                   if(sc.Days[i].sessions[x].id==session){
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Expected '===' and instead saw '=='.](https://www.codacy.com/app/triplez/open-event-webapp/file/2406535184/issues/source?bid=3233857&fileBranchId=3233878#l89)